tests/jobs/test_local_task_job.py

```diff
@@ -0,0 +1,203 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+import logging
+import multiprocessing
+import time
+import unittest
+
+from airflow import AirflowException, models, settings
+from airflow import configuration
+from airflow.executors import SequentialExecutor
+from airflow.jobs import LocalTaskJob
+from airflow.models import DAG, TaskInstance as TI
+from airflow.operators.dummy_operator import DummyOperator
+from airflow.task.task_runner.base_task_runner import BaseTaskRunner
+from airflow.utils import timezone
+from airflow.utils.net import get_hostname
+from airflow.utils.state import State
+from tests.compat import patch
+from tests.core import TEST_DAG_FOLDER
+from tests.test_utils.db import clear_db_runs
+
+configuration.load_test_config()
+
+logger = logging.getLogger(__name__)
```

Comment:
I don't believe the `logger` is used in this script, is that necessary here?

Reply:
Looks like it is not used. I removed it.